target/linux/ath79/dts/ar9344_netgear_pgzng1.dts

```diff
@@ -0,0 +1,341 @@
+				type = <PCA955X_TYPE_GPIO>;
+			};
+		};
+
+		rtc@6f {
+			compatible = "isil,isl1208";
+			reg = <0x6f>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		eth1_link {
+			label = "green:eth1";
+			gpios = <&gpio 0 GPIO_ACTIVE_HIGH>;
+		};
+
+		eth1_act {
+			label = "amber:eth1";
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+		};
+
+		eth0_link {
+			label = "green:eth0";
+			gpios = <&gpio 2 GPIO_ACTIVE_HIGH>;
+		};
+
+		eth0_act {
+			label = "amber:eth0";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&eth0 {
+    status = "okay";
+	phy-handle = <&swphy0>;
+
+	nvmem-cells = <&macaddr_caldata_0>;
+	nvmem-cell-names = "mac-address";
+};
+
+&eth1 {
+    status = "okay";
+    phy-handle = <&swphy4>;
+
+	nvmem-cells = <&macaddr_caldata_6>;
+	nvmem-cell-names = "mac-address";
+
+    gmac-config {
+        device = <&gmac>;
+        switch-phy-swap = <1>;
+        switch-only-mode = <1>;
+    };
+};
+
+&gpio {
+	gpio_ext_lna0 {
+		gpio-hog;
+		line-name = "ext:lna0";
+		gpios = <18 GPIO_ACTIVE_HIGH>;
+		output-high;
+	};
+
+	gpio_ext_lna1 {
+		gpio-hog;
+		line-name = "ext:lna1";
+		gpios = <19 GPIO_ACTIVE_HIGH>;
+		output-high;
+	};
+
+	gpio_enable_rs422 {
+		gpio-hog;
+		line-name = "power:rs422";
+		gpios = <20 GPIO_ACTIVE_HIGH>;
+		output-high;
+	};
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x40000>;
+			read-only;
+		};
+
+		partition@40000 {
+			label = "u-boot-env";
+			reg = <0x40000 0x40000>;
+			read-only;
+		};
+
+		/* factory kernel used to be reg = <0x80000 0x200000> */
+		partition@80000 {
+			label = "kernel";
+			reg = <0x80000 0x500000>;
+		};
+
+		/* factory rootfs used to be reg = <0x280000 0x5000000> */
+		partition@580000 {
+			label = "ubi";
+			reg = <0x580000 0x4d00000>;
+		};
+
+		partition@5280000 {
+			label = "uImage2";
+			reg = <0x5280000 0x200000>;
+			read-only;
+		};
+
+		partition@5480000 {
+			label = "rootfs_bak";
+			reg = <0x5480000 0x5000000>;
+			read-only;
+		};
+
+		partition@a480000 {
+			label = "config";
+			reg = <0xa480000 0x1400000>;
+			read-only;
+		};
+
+		partition@b880000 {
+			label = "storage";
+			reg = <0xb880000 0x4700000>;
+			read-only;
+		};
+
+		partition@ff80000 {
+			label = "dummy";
+			reg = <0xff80000 0x60000>;
+			read-only;
+		};
+
+		caldata: partition@ffe0000 {
+			label = "caldata";
+			reg = <0xffe0000 0x20000>;
+			read-only;
+
+			compatible = "nvmem-cells";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			caldata_1000: cal@1000 {
+				reg = <0x1000 0x440>;
+			};
+		};
+	};
+};
+
+&ref {
+	clock-frequency = <40000000>;
+};
+
+/* zWave is wired up via SPI and UART1 (no idea on pins sadly) */
+&spi {
+	status = "okay";
+};
+
+&wmac {
+	status = "okay";
+
+	nvmem-cells = <&caldata_1000>;
+	nvmem-cell-names = "calibration";
+};
+
+&usb {
+	status = "okay";
+};
+
+&usb_phy {
+	status = "okay";
+};
+
+&caldata {
```

Comment:
please integrate this into the caldata node from above. (you can drop the compatible and #address-cells + #size-cells since they are specified twice)